tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "lib": ["es2017"],
+        "lib": ["es2016"],
```

Comment:
As discussed in #3278, we use a polyfill for `Object.values` and `Object.entries`, which are `es2017` features. We also don't plan to use TypeScript as a transpiler.